index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1 class="title">Hello World, we are software developers</h1>
+
+    <p id="description"></p>
+
+    <form>
+  <label id="name-label">Name:</label><br>
+  <input type="text" id="name" value="Enter your full name"><br>
+  <label id="email-label">Email:</label><br>
+  <input type="email" id="email" value="example@email.com"><br>
+  <label>Number:</label><br>
+  <input type="number" id="number" name="number" min="10" max="99" value="0900000000">
+  
+  
+</form>
+
+
+
+
+
+<h3>Which option best describes your current role?</h3>
+<select name="" id="dropdown">
+ 
+  <option disabled selected value>Select current role</option>
+      <option value="a"  >developer</option>
+      <option value="a">student </option>
+      <option value="a">worker</option>
+</select>
+<h3>Would you recommend Zoum-Center to a friend?</h3>
+<div class="form-input">
+  <label for="d"> <input name="user-recommend"value="Definitely"
+ type="radio"
+    class="input-radio">Definitely </label>
+  <label for="d"> <input name="user-recommend"
+    value="maybe"
+    type="radio"
+    class="input-radio">Maybe </label>
+  <label for="d"> <input name="user-recommend"
+    value="Not sure"
+    type="radio"
+    class="input-radio"> Not sure</label>
+</div>
+
+
+<div class="form-group">
+  <p>What is your favorite feature of Zoum-Center ?</p>
+  <select id="most-like" name="mostLike" class="form-control" required>
+    <option disabled selected value>Select an option</option>
+    <option value="challenges">Challenges</option>
+    <option value="projects">Projects</option>
+    <option value="community">Community</option>
+    <option value="openSource">Open Source</option>
+  </select>
+</div>
+    
+
+
+
```

Comment:
Optional suggestions

Please consider removing all the unused/empty lines from your code base to make it more professional as unused lines of code is a cause of distraction.